mvvm/src/iosMain/kotlin/dev/icerock/moko/mvvm/binding/UILabelBinding.kt

```diff
@@ -6,29 +6,18 @@ package dev.icerock.moko.mvvm.binding
 
 import dev.icerock.moko.mvvm.livedata.LiveData
 import dev.icerock.moko.mvvm.livedata.map
-import dev.icerock.moko.resources.desc.StringDesc
 import platform.UIKit.UILabel
 import dev.icerock.moko.mvvm.utils.bind
 
-fun UILabel.bindText(
-    liveData: LiveData<String>,
+fun <T> UILabel.bindText(
+    liveData: LiveData<T>,
     formatter: ((String) -> String) = { it }
 ) {
```

Comment:
it allow use `bindText` with any livedata, as `LiveData<Image>` - it not correct